monero-sys/src/bridge.h

```diff
@@ -300,6 +333,28 @@ namespace Monero
         return vec;
     }
 
+    inline bool addSubaddress(Wallet* wallet, uint32_t account_index, const std::string &label)
+    {
+        try {
+            auto* sub = wallet->subaddress();
+            sub->addRow(account_index, label);
+            return true;
+        } catch(...) {
+            return false;
```

Comment:
We don't want to catch error here as this means we silently fail and lose the error context